hashing/te-tag-query-java/APIExamples.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+// ================================================================
+// The main reference tooling is com/facebook/threatexchange/TETagQuery.java,
+// which is likely most useful. However, this file is a more clear-cut example
+// of incorporating the API into your own framework.
+//
+// Compile with
+//
+//   javac com/facebook/threatexchange/*.java APIExamples.java
+//
+// and run with
+//
+//   java APIExamples
+// ================================================================
+
+import com.facebook.threatexchange.Net;
+import com.facebook.threatexchange.DescriptorPostParameters;
+
+import java.io.IOException;
+
+public class APIExamples {
+  // ----------------------------------------------------------------
+  public static void main(String[] args) throws IOException {
+    Net.setAppToken("TX_ACCESS_TOKEN");
+    submitExample();
+    updateExample();
+    copyExample();
+  }
+
+  // ----------------------------------------------------------------
+  private static void submitExample() {
+    boolean showURLs = false;
+    boolean dryRun = false;
+    DescriptorPostParameters postParams = new DescriptorPostParameters();
+
+    postParams.setIndicatorText("dabbad00f00dfeed5ca1ab1ebeefca11ab1ec00e");
+    postParams.setIndicatorType("HASH_SHA1");
+    postParams.setDescription("testing API-example post");
+    postParams.setShareLevel("AMBER");
+    postParams.setStatus("NON_MALICIOUS");
+    postParams.setPrivacyType("HAS_WHITELIST");
+    postParams.setPrivacyMembers("1064060413755420");
```

Comment:
I think privacyMembers should definitely have a comment explaining it and maybe linking to the docs. At least to note that the id provided is a test app. Aside: also if a user wanted to specify more than one privacy group/member does just adding a coma between them work?
(https://developers.facebook.com/docs/threat-exchange/reference/privacy)

Reply:
nice